src/scribe_data/cli/get.py

```diff
@@ -100,40 +103,56 @@ def get_data(
 
     # MARK: Get All
     if all:
-        if language:
-            language_or_sub_language = language.split(" ")[0]
-            print(f"Updating all data types for language for {language.title()}")
-            query_data(
-                languages=[language_or_sub_language],
-                data_type=None,
-                output_dir=output_dir,
-                overwrite=overwrite,
-            )
-            print(
-                f"Query completed for all data types with specified language for {language.title()}."
-            )
-
-        elif data_type:
-            print(f"Updating all languages for data type: {data_type.capitalize()}")
-            query_data(
-                languages=None,
-                data_type=[data_type],
-                output_dir=output_dir,
-                overwrite=overwrite,
-            )
-            print(
-                f"Query completed for all languages with specified data type for {data_type.capitalize()}."
-            )
+        # Using wikimedia lexeme based dump
 
+        if wiki_dump:
+            print("wiki_dump", wiki_dump)
+            download_wrapper(None, wiki_dump)
         else:
-            print("Updating all languages and data types...")
-            query_data(
-                languages=None,
-                data_type=None,
-                output_dir=output_dir,
-                overwrite=overwrite,
-            )
-            print("Query completed for all languages and all data types.")
+            print("Using wikimedia lexeme dump...")
+            file_path = download_wrapper()
+            if isinstance(file_path, str) and file_path:
+                rprint("[bold green]we'll use this lexeme dump[/bold green]", file_path)
+                rprint(
+                    "[bold red]Parsing lexeme dump feature will be available soon...[/bold red]"
+                )
+
+        # Using sparql based data extract
```

Comment:
Very long story 😅

We're not moving away from SPARQL at all. I had a recent interview with some coworkers where I talked about the planned functionality of Scribe-Data, and the general sentiment was "Cool, yes, but why are you planning on sending that many queries to the Wikidata query service even on a bi-weekly basis." Even now we're talking ten minutes of queries, and at scale it's going to be tons of very long queries.

The plan from here (feedback welcome!):
1. Scribe-Data develops a Wikidata dump process, not a Wiktionary dump process, as "there's a chance that lots of translations will be added soon". There are also benefits of this regardless - see below.
2. Scribe-Server still runs Scribe-Data, but uses the most recent dump such that the query load is 0 and the only load on the WMF side is transferring the dump (the query service is strained, the download process is not).
3. Scribe-Data thus has dump and SPARQL processes that mirror one another
    - Results are the same for each
    - A dump is required for `scribe-data get -a`
        - The user can still do `scribe-data get -a` with queries if the want via looping through Scribe-Data commands, but for responsible use purposes it requires a dump in the base command
    - Using a dump is suggested for `scribe-data get -lang LANGUAGE -a` and `scribe-data get -dt DATA_TYPE -a`, but if the user wants to run the queries then they can
    - For commands like 
4. The beauty of the above is that it allows us to directly solve #513
    - We create a workflow to download the most recent dump and extract `__all__` forms from it
    - We check the forms that we're getting from the dump against the current queries
    - If there are forms that we don't have, we trigger a pull request to Scribe-Data that leverages the [src/scribe_data/resources/lexeme_form_metadata.json](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/resources/lexeme_form_metadata.json) file - needs to be updated as more property IDs are needed - that says "Hey couldn't help but notice your LANGUAGE query for DATA_TYPE doesn't have MISSING_FORMS - here's the query as I've used `lexeme_form_metadata` to translate the Wikidata PIDs into appropriate labels"
    - We thus have a process where the queries are automatically updated to get all data so that people can use them to get the data or the dumps if they so choose

As stated, long story 😊 Let me know on the above!